driver-sync/src/test/functional/com/mongodb/client/AbstractUnifiedTest.java

```diff
@@ -94,15 +96,22 @@
     private Map<String, BsonDocument> lsidMap;
     private boolean useMultipleMongoses = false;
     private ConnectionString connectionString = null;
-    private final String collectionName = "test";
+    private final String collectionName;
+    private MongoDatabase database;
 
     private static final long MIN_HEARTBEAT_FREQUENCY_MS = 50L;
 
     public AbstractUnifiedTest(final String filename, final String description, final BsonArray data, final BsonDocument definition,
                                final boolean skipTest) {
+        this(filename, description, getDefaultDatabaseName(), "test", data, definition, skipTest);
```

Comment:
The new write concern tests have close to the same format as the transactions tests, so the AbstractUnifiedTest class needed some changes to support the new tests. One change is that the tests specify the collection name to be used. So I have created two constructors, one to specify a default collection name and the other to set a collection name.